Evaluate-ModelarDB-Changes/main.py

```diff
@@ -160,24 +152,38 @@ def send_sigint_to_process(process):
 
     process.wait()
 
+    stderr = process.stderr.read()
+    if errors_occured(stderr):
+        print_stream(stderr)
+        return None
+    else:
+        # Indicate no errors occurred.
+        return True
+
 
 def append_finished_result(
-    output_file, changes, ingestion_time, query_time, data_folder_size
+    output_file, current_change, changes, ingestion_time, query_time, data_folder_size
 ):
     results = {
+        "changes": changes,
         "ingestion_time_in_seconds": ingestion_time,
         "query_time_in_seconds": query_time,
         "data_folder_size_in_kib": data_folder_size,
     }
 
     output_file.write('  "')
-    output_file.write(" ".join(changes))
+    output_file.write(str(current_change))
     output_file.write('": ')
     output_file.write(json.dumps(results))
     output_file.write(",\n")
     output_file.flush()
 
 
+def print_seperator(current_change, last_change):
```

Comment:
Minor but I think it should be `separator` instead.

Reply:
Fixed.